Sources/CALayer.swift

```diff
@@ -19,11 +19,7 @@ open class CALayer {
     /// the view sets this value to match the screen.
     open var contentsScale: CGFloat = 1.0
 
-    internal var contentsGravityEnum: ContentsGravity = .resize
-    open var contentsGravity: String {
-        get { return contentsGravityEnum.rawValue }
-        set { contentsGravityEnum = ContentsGravity(rawValue: newValue) ?? .center } // matches iOS
-    }
+    open var contentsGravity: CALayerContentsGravity = .resize
```

Comment:
I think this `.center` fallback behaviour is still relevant, probably in `CALayerContentsGravity(rawValue: String)`

Reply:
like this? feels very strange:
f0b41fe33c9298fda07238706b6022cb351e324a

Reply:
Since we implement `CALayerContentsGravity` with an `enum` and it's `init(rawValue)` is failable, we don't need the fallback anymore. The API consumer is not able to do anything stupid when setting `CALayer.contentsgravity` via `init(rawValue:)?`, because he has to provide a fallback himself.

Reply:
You're right, but there may well be users who rely on `init(rawValue:)`, meaning we should implement it too, including the weird fallback. It would be nicer if Apple just used an enum like we did, but it's very likely because their UIKit is in ObjC.

We can (and should) still use an enum ourselves, but with `: String` to add the initializer